### PR TITLE
docs: add `Nuxt: A vision for 2023` post

### PR DESCRIPTION
**Pull request by danielroe**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This adds an announcement post: `Nuxt: A vision for 2023`.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by toniengelhardt** [comment]
Would be amazing to clarify the path forward for the `auth` and `pwa` modules, aka. if [nuxt-pwa-module](https://github.com/kevinmarrec/nuxt-pwa-module) is going to be the official PWA module or if there will be a new one (it looks like Elk is not using it), and also if there is already an experimental repo for the auth module somewhere or if there will be something new. 🙏🏽

File: docs/content/vision-2023.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    link: https://twitter.com/danielcroe
+date: "Jan 17, 2023"
+---
+#title
+# Nuxt: A vision for 2023
+#description
+This past year has been an exciting one. Looking into the new year, there is a lot we have planned as a team and we'd love to share it with you. :sparkles:
+#body
+
+This past year has been an exciting one, with the release of Nuxt 3 and Nitro and the launch of the new [nuxt.com](http://nuxt.com/) website. It's been the culmination of years of work, and has not only resulted in a new major version of Nuxt, but a new Nuxt architecture, a full-stack server framework ([Nitro](https://nitro.unjs.io/)), and a new GitHub organisation and ecosystem ([unjs](https://github.com/unjs/)).
```
Comment (DamianGlowala) [inline docs/content/vision-2023.md:22]:
```suggestion
This past year has been an exciting one, with the release of Nuxt 3 and Nitro and the launch of the new [nuxt.com](http://nuxt.com/) website. It's been the culmination of years of work, and has not only resulted in a new major version of Nuxt, but a new Nuxt architecture, a full-stack server framework ([Nitro](https://nitro.unjs.io/)), and a new GitHub organisation and ecosystem ([UnJS](https://github.com/unjs/)).
```

File: docs/content/vision-2023.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Throughout that whole time, Pooya Parsa has led the Nuxt team, putting in countless hours of work and thought into building Nuxt 3.
+
+Now, at the start of 2023, he's handing over the helm of the Nuxt open-source work to me ([Daniel Roe](https://github.com/danielroe)). Pooya will continue to be actively contributing to the Nuxt project and of course driving the development of UnJs ecosystem and Nitro project.
```
Comment (DamianGlowala) [inline docs/content/vision-2023.md:26]:
```suggestion
Now, at the start of 2023, he's handing over the helm of the Nuxt open-source work to me ([Daniel Roe](https://github.com/danielroe)). Pooya will continue to be actively contributing to the Nuxt project and of course driving the development of UnJS ecosystem and Nitro project.
```

File: docs/content/vision-2023.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+The main priorities at the start of the year are `nuxt/image`, PWA and `nuxt/auth`.
+
+We're also developing RFCs for `nuxt/font` and `nuxt/script` in conjunction with the Google Aurora team, which will make it much easier to apply best performance practices to your Nuxt apps. Watch this space!
+
+## DX and Performance
+
+We particularly care a lot about making Nuxt a joy to use, and we'd like to keep pushing the boundary of great developer experience, which we believe results in the best experience for users of the apps we write too.
+
+In the coming months, there will be a continued focus on developer experience and performance. Expect to see Nuxt DevTools and CLI improvements for scaffolding - and more. On the performance side, Nuxt 3 + Nitro is a game-changer for speed, performance, and customisability, and we’ll be building on top of that to enable some amazing features. 🚀
```
Comment (DamianGlowala) [inline docs/content/vision-2023.md:66]:
```suggestion
In the coming months, there will be a continued focus on developer experience and performance. Expect to see Nuxt DevTools and CLI improvements for scaffolding - and more. On the performance side, Nuxt 3 + Nitro is a game-changer for speed, performance, and customizability, and we’ll be building on top of that to enable some amazing features. 🚀
```

File: docs/content/vision-2023.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- experimental server-only components and a component island renderer
+- Nitro 2, Vite 4 and Rollup 3 support
+
+## Migrating to Nuxt 3
+
+On December 31st, 2023, Vue 2 will reach End of Life (EOL), and with it Nuxt 2. Both Vue and Nuxt will continue being available and working for many people, but at the same time, many companies and users will want to transition to Nuxt 3 (and we'd encourage this!).
+
+Part of our focus this year therefore will be supporting everyone who wants to migrate their apps to Nuxt 3. We'll also be working to backport key bug fixes and security fixes to Nuxt 2.
+
+In addition, there is Nuxt Bridge. It was built as a module to bring features and bug fixes from Nuxt 3 back to Nuxt 2, although it has not yet been released in a stable version. We plan to stabilise and release it in the next month or so, but our main focus over the course of the year will be on helping people transition to Nuxt 3.
```
Comment (DamianGlowala) [inline docs/content/vision-2023.md:87]:
```suggestion
In addition, there is Nuxt Bridge. It was built as a module to bring features and bug fixes from Nuxt 3 back to Nuxt 2, although it has not yet been released in a stable version. We plan to stabilize and release it in the next month or so, but our main focus over the course of the year will be on helping people transition to Nuxt 3.
```

**Review by DamianGlowala — DISMISSED** [review]

File: docs/content/vision-2023.md (hunk 3 above)
Comment (danielroe) [reply docs/content/vision-2023.md:66]: I'll keep this one. 😊

**Review by Atinux — APPROVED** [review]

**Comment by Aareksio** [comment]
Very glad to see migration path addressed! Wish you the best in new role!

**Pull request merged**